docs/knowledgebase/advanced/block-import.md

```diff
@@ -4,33 +4,31 @@ title: The Block Import Pipeline
 
 ## The Import Queue
 
-The import queue is an abstract worker queue present in every Substrate node. It is not part of the
-runtime. The import queue is responsible for processing pieces of incoming information, verifying
-them, and if they are valid, importing them into the node's state. The most fundamental piece of
-information that the import queue processes is blocks themselves, but it is also responsible for
-importing consensus-related messages such as justifications and, in light clients, finality proofs.
+The import queue is the abstract worker queue present in every Substrate node. The import queue is responsible for processing pieces of incoming information, verifying validity and importing valid information into the node's state. The most fundamental piece of
+information that the import queue processes is blocks. It is also responsible for
+importing consensus-related messages such as justifications or in light clients, finality proofs.
 
 The import queue collects incoming elements from the network and stores them in a pool. The elements
 are later checked for validity and discarded if they are not valid. Elements that are valid are then
 imported into the node's local state.
 
 The import queue is codified abstractly in Substrate by means of the
-[`ImportQueue` trait](https://substrate.dev/rustdocs/v2.0.0-rc4/sp_consensus/import_queue/trait.ImportQueue.html).
+[`ImportQueue` trait](https://substrate.dev/rustdocs/v2.0.0-rc5/sp_consensus/import_queue/trait.ImportQueue.html).
 The use of a trait allows each consensus engine to provide its own specialized implementation of the
-import queue, which may take advantage of optimization opportunities such as verifying multiple
+import queue, which optimizes by verifying multiple
```

Comment:
There are currently no import queues implemented that do this parallel verification. This is an example of the kind of thing that an import queue _might_ do and motivates why we use a Trait.